go/teams/loader_types.go

```diff
@@ -21,7 +21,8 @@ type chainLinkUnpacked struct {
 	source    *SCChainLink
 	outerLink *libkb.OuterLinkV2WithMetadata
 	// inner is nil if the link is stubbed
-	inner *SCChainLinkPayload
+	inner  *SCChainLinkPayload
+	linkID libkb.LinkID
```

Comment:
This linkID is only non-blank if the link is not stubbed, unlike `.LinkID()`. It's creepy to have blank data sitting around. How about just calculate it in AssertInnerOuterMatch? Or assert assert a match in `unpackChainLink` if it's not stubbed.